client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -341,3 +342,19 @@ export function getArrayOfFilteredTasks( tasks: Task[], flow: string | null ) {
 		}, [] as Task[] )
 	);
 }
+
+// Returns enhanced task list with domain_upsell task removed
+// Only applies to Free, Write & Build flow sites with paid plan
+export function filterDomainUpsellTask(
```

Comment:
Is there a specific reason we moved this method into the `task-helper` file? I generally try and avoid adding a layer of separation between code and where it's used when possible.

There's less cognitive load when you're able to see everything in a single file

Reply:
I wanted to make it easier to test that function specifically and also didn't want to add all that logic into the component. I could move it into the sidebar component if it makes it easier to understand and helps with readability.

Reply:
> I could move it into the sidebar component if it makes it easier to understand and helps with readability.

I'd appreciate a follow-up, but not necessary if there's other high pri work